app/controllers/recipe_ingredients_controller.rb

```diff
@@ -0,0 +1,7 @@
+class RecipeIngredientsController < ApplicationController
+  def create
+    RecipeIngredient.create!(recipe: Recipe.find(params[:ingrediant_id]), ingredient: Ingredient.find_by(name: params[:name]))
```

Comment:
This is a great start. I think you could make it a little simpler with something like:
```ruby
RecipeIngredient.create!(recipe_id: params[:id], ingredient_id: params[:ingredient_id])
```
Note you will see what params you have in `pry`. Then your redirect would be to `"/recipes/#{params[:id]}"` (or whatever param is storing your recipe id.)